src/app/page/UserProfile/UserOverview.jsx

```diff
@@ -50,7 +50,7 @@ const UserStatus = ({ profile, setProfile, view }) => {
       type: ["(cities)"],
     };
     const addrElem = document.getElementById("address");
-    const googleRef = !!window && !!window.google ? window.google : undefined;
+    const googleRef = window.google;
```

Comment:
Without the ternary logic, I think this is an unnecessary variable assignment and it would be more clear to just explicitly use `window.google` instead of `googleRef`.